lib/ex_unit/lib/ex_unit/runner_stats.ex

```diff
@@ -25,6 +25,11 @@ defmodule ExUnit.RunnerStats do
     {:noreply, %{map | total: total + 1, skipped: skipped + 1}}
   end
 
+  def handle_cast({:case_finished, %ExUnit.TestCase{state: {:failed, _reason}}}, %{failures: failures, totals:
+  totals} = map) do
+    {:noreply, %{map | failures: failures + 1, totals: totals + 1}}
```

Comment:
Just to clarify (and with the strong proviso that I'm pretty much fumbling my way through the macros here), it looks like I'd be calling `length(case_name.__ex_unit__(:case).tests)`. Does that seem right?

Another clarification: A raise in a `setup_all`will fail all the tests in that case, so I'm assuming that we'd be doing the same sort of math with the failures.

Basically, in all ways, this should mirror the behavior of a single test failing (except all the tests in the case fail) for the purposes of reporting, correct?